src/metrics.md

```diff
@@ -24,34 +24,34 @@ The following abbreviations are used throughout this page and site:
 * SP - storage provider.  And by "storage provider", we mean the entity as defined in the Filecoin protocol with an individual id that commits sectors, accepts deals, etc.
 
 ## Aggregate Metrics
-These are metrics looking across all stroage providers in the network.  They correspond with the aggregate views displayed at the [dashboard home](/).
+These are metrics looking across all storage providers in the network.  They correspond with the aggregate views displayed at the [dashboard home](/).
 
 ### Providers Making State Market Deals
 This looks at the number of providers who made a state market deal on a given day.  This gives a sense of the number of "active storage providers".
 
 ### Providers With Active State Market Deals
-On a given day, how many storage providers have a active state market deals.  This is gives a sense of the number of "active storage providers".
+On a given day, how many storage providers have a active state market deals.  This gives a sense of the number of "active storage providers".
 
 ### "Warm" Service Class Conformance
 This shows the number of Storage Providers in a given day that met the ["warm" service class service level objectives](https://github.com/filecoin-project/service-classes/service-classes/warm.md).  If a given SP only meets some of the SLOs, these are highlighted as well.
 
 ### "Cold" Service Class Conformance
-This shows the number of Storage Providers in a given day that met the ["cold" service class service level objectives](https://github.com/filecoin-project/service-classes/service-classes/cold.md).  If a given SP only meets some of the SLOs, these are highlighted as well. 
+This shows the number of Storage Providers in a given day that met the ["cold" service class service level objectives](https://github.com/filecoin-project/service-classes/service-classes/cold.md).  If a given SP only meets some of the SLOs, these are highlighted as well.
 
 ## Individual Storage Provider Metrics
 These are metrics that specific to an individual SP at `/provider/providerId`.
 
 ### Spark Retrieval Success Rate
-This shows a Storage Provider's [Spark Retrieval Success Rate](https://github.com/filecoin-project/service-classes/service-level-indicators/spark-retrieval-success-rate.md).  The horizontal line's value, which corresponds with a service level objective, is defined in the supporting [service class definition](https://github.com/filecoin-project/service-classes/service-classes) (e.g., ["warm"](https://github.com/filecoin-project/service-classes/service-classes/warm.md)).
```

Comment:
I think "e.g.," is right.  https://www.scribbr.com/definitions/for-example-abbreviation/

Reply:
Interesting! TIL. Will update the linter rules :sweat_smile: